core/unix/rseq_linux.c

```diff
@@ -77,14 +77,11 @@ typedef struct _rseq_region_t {
     app_pc end;
     app_pc handler;
     /* We need to preserve input registers for targeting "start" instead of "handler"
-     * for our 2nd invocation.  We only support GPR inputs.
+     * for our 2nd invocation, if they're written in the rseq region.  We only support
+     * GPR inputs.  We document that we do not support any other inputs (no flags, no
```

Comment:
s/inputs/outputs/

Reply:
They're not outputs: if they're written, we do not care about the *new* value: we have to preserve the *old* value.